tmt/result.py

```diff
@@ -187,7 +187,24 @@ class CheckResult(BaseResult):
     event: CheckEvent = field(
         default=CheckEvent.BEFORE_TEST,
         serialize=lambda event: event.value,
-        unserialize=CheckEvent.from_spec)
+        unserialize=CheckEvent.from_spec
+        )
+
+    def interpret_check_result(self, interpret: CheckResultInterpret) -> 'CheckResult':
+        """Interpret check result according to the check_result interpretation."""
```

Comment:
The docstring format should probably be changed to:

```
""" Interpret check result according to the check_result interpretation """
```

or even to:

```
""" Interpret check result according to the ``check_result`` interpretation """
```

Reply:
There's no ``check_result`` in the docstring or signature :) Maybe "check result" would be better?

Reply:
changed to "check_result" in https://github.com/teemtee/tmt/pull/3239/commits/345bde830384b610d0d4bccd395b7f0a00aa5b26

Reply:
my bad, I forgot the underscore.

Reply:
oh, after actually thinking about it - I should also drop the quotes, eh? :)

Reply:
Yep :)